kotlin/codegen/pom.xml

```diff
@@ -47,6 +47,11 @@
       <artifactId>kotlinpoet-classinspector-elements</artifactId>
       <version>${kotlinpoet.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.ow2.asm</groupId>
```

Comment:
Much of the asm bits used are derived from prior work done in the unfinished bridge generator: https://github.com/square/moshi/blob/z/generateBytecode/kotlin/codegen/src/main/java/com/squareup/moshi/kotlin/codegen/BridgeGenerator.kt